modules/install-central-operator-external-db.adoc

```diff
@@ -18,7 +18,7 @@ For more information about {product-title-short} databases, see the link:https:/
 
 .Prerequisites
 * You must be using {ocp} {ocp-supported-version} or later. For more information about supported {ocp} versions, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix].
-* You must have a database in your database instance that supports PostgreSQL 13 and a user with the following permissions:
+* You must have a database in your database instance that supports PostgreSQL 15 and a user with the following permissions:
```

Comment:
Similar as above.  Supports Postgres 13 or 15.

Primary reason for all this is ACSCS is going to be running 13 for quite some time.